src/handlers/no_merges.rs

```diff
@@ -48,11 +48,13 @@ pub(super) async fn parse_input(
         return Ok(None);
     }
 
-    // Don't trigger if the PR has any of the excluded labels.
-    for label in event.issue.labels() {
-        if config.exclude_labels.contains(&label.name) {
-            return Ok(None);
-        }
+    // Don't trigger if the PR has any of the excluded title segments.
+    if config
+        .exclude_titles
+        .iter()
+        .any(|s| event.issue.title.contains(s))
```

Comment:
Any particular reason to not make this case-insensitive? Seems like making it insensitive could make it a little less likely to have false-positives.

Reply:
Obviously it makes the implementation more complicated, but the real reason is that it reduces specificity.

Capitalization can be used to disambiguate. For example, on `rust-clippy`, using the capitalized "Rustup" disambiguates between [PRs updating from `rust`](https://github.com/rust-lang/rust-clippy/pull/11469) and ones [just mentioning "rustup"](https://github.com/rust-lang/rust-clippy/pull/7239)

Since only the first letter in a word is capitalized, generally it only takes a couple more entries to exclude the various combinations if so desired.